src/libNode/Node.cpp

```diff
@@ -134,6 +134,11 @@ bool Node::Install(const SyncType syncType, const bool toRetrieveHistory) {
       return false;
     }
 
+    if (SyncType::NEW_SYNC == syncType ||
+        SyncType::NEW_LOOKUP_SYNC == syncType) {
+      return true;
+    }
```

Comment:
If we don't skip what will happen?

Reply:
Haven't tried.. Just that we don't need the code below for new nodes joining.